setup/install/providers/aws/aws-ec2.md

```diff
@@ -85,22 +91,31 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
 --parameter-overrides AuthArn=FROM_ABOVE ManagingAccountId=FROM_ABOVE --capabilities CAPABILITY_NAMED_IAM --region us-west-2
 ```
 
-## Option-3 : Use AWS Console UI (Manual Steps) 
+## Option-3 : Configure with AWS IAM Console
+### Create Roles
+
+
+### Create Policies
+
+
+### Associate Policies to User or Roles
+The permissions defined in the policies are attached to the users or roles. This will allows Spinnaker to use either a user or a role and deploy EC2 instances in any particular region.
 
 There are 2 options here
-1. Using AWS IAM AccessKey and Secret
+1. Using AWS IAM User with AccessKey and Secret
 Option number 1 is useful for creation of user with AWS Access Key and secret. This is a common configuration. 
 2. Using AWS IAM Roles
 Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
 
 ## Halyard Configurations
-After the AWS IAM user, roles, policies and trust relationship have been set up, the next step is to add the AWS configurations to Spinnaker via Halyard CLI:
+After the AWS IAM  structure (user, roles, policies and trust relationship) has been set up, the next step is to add the AWS configurations to Spinnaker via Halyard CLI:
 
+The General steps are the following:
 1. Access the Halyard Pod.
 2. Add the configurations for AWS provider with `hal` command. Please check [hal config provider AWS](https://www.spinnaker.io/reference/halyard/commands/#hal-config-provider-aws).
 3. Enable the AWS provider `hal config provider aws enable`.
 
-### Configure Halyard to use AccessKeys (if configured)
+### Configure Spinnaker AWS provider to use AccessKeys (if using AWS IAM user)
```

Comment:
I like these changes a lot, @aleon1220 ! You don't "configure Halyard," you use Halyard to configure Spinnaker!